README.md

```diff
@@ -166,95 +152,23 @@ gem 'pg'
+Note: You can also `$ gem install taperole` and not put Taperole in your 
+Gemfile.
 
-### Provisioning/Deploying a Rails Application server with Taperole
+During your first deploy, your app will not have a `secrets.yml` file configured, and Tape will prompt you to provide one:
 
-1. Confirm that `taperole`, `pg`, and `unicorn` are in your gemfile.
-2. `$ bundle`
-3. When asked `Are you going to use vagrant? (y/n):`, say `n`.
-4. Make your hosts file. Assuming you're going to have multiple environments, it should look something like this:
-```
-[production]
-0.0.0.0 be_app_env=production be_app_branch=master
-
-[omnibox:children]
-production
-```
-5. Give yourself access to the server. This needs to be the same ssh key that you used on your digital ocean droplet. So create a file called <your_name>.pub in the dev_keys folder that was generated by tape installer install. `pbcopy < ~/.ssh/id_rsa.pub` in your terminal again, then paste the result into the file you just created.
-6. Update your tape_vars.yml file.
-```
-app_name: walkthrough
-
-be_app_repo: [git repo]
-
-dev_key_files:
-  - dev_keys/your_name.pub
-```
-7. Copy all developers public keys into the `dev_keys` directory.
-8. Add this to your gitignore.
-```
-# Ignore this stuff for Taperole
-config/secrets.yml
-config/database.yml
-```
-9. If it isn't automatically, comment out front end stuff:
- * `# fe_deploy` in omnibox.yml
- * `# - frontend_deploy` in deploy.yml
-
-10. git add and commit and push to master
-11. Run `$ tape ansible everything.` This will take a long time, so grab a sandwich or something.
-12. This should chug right along until it gets to
 ```
 TASK: [backend_config | Ask for secrets.yml] **********************************
 ok: [159.203.126.223] => {
     "msg": "You've got to upload secrets.yml to /home/deployer/kevinrkiley/config to continue"
 }
 ```
-where it will hang. Don't cancel the deploy.
-13. Open a new terminal tab and generate a secret key by running `$ rake secret` from the root directory of your rails app. Don't lose this.
-14. Run `$ ssh deployer@<ip_to_server>`
-15. Once you're in the server, `$ cd walkthrough/config`. If you `$ ls`, you should see that there is no `secrets.yml` file. `$ vi secrets.yml` to create it.
-16. When you're done secrets.yml should look something like:
-```
-production:
-  secret_key_base: thisisthesecretkeyyougeneratedinstep9
-```
-17. Your deployment should automatically continue.
-
-
-### Deploying your latest changes
-
-To deploy your latest changes to your tape-provisioned server:
-
-```
-tape ansible deploy
-```
-
-This will git pull the latest changes from the tracking branch you specified and restart all services via monit.
 
-This command runs all Ansible roles specified in the deploy.yml playbook.
+To continue the deploy, SSH into the server as the `deployer` user and create your `config/secrets.yml` file in the app directory. The deploy will automatically continue when the file is saved.
 
 ### Rake tasks
 
```

Comment:
I think this functionality is actually not working anymore. I broke it when I rewrote the run layer. We might want to delete this section and create a new issue for fixing that.